### PR TITLE
Chore: Adding interaction targets

### PR DESCRIPTION
**Pull request by jeremypress**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #276

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentSidebar/ContentSidebar.js [hunk 1]
```diff
@@ -423,7 +423,11 @@ class ContentSidebar extends PureComponent<Props, State> {
 
         return (
             <Internationalize language={language} messages={intlMessages}>
-                <aside id={this.id} className={`be bcs ${className}`}>
+                <aside
+                    id={this.id}
+                    data-resin-component={CLIENT_NAME_CONTENT_SIDEBAR.toLowerCase()}
```
Comment (priyajeet) [inline src/components/ContentSidebar/ContentSidebar.js]: remove this, we are only adding a target

**Comment by priyajeet** [comment]
Don't push this one out, since the skills structure has changed.
You can work on the other resin stuff, just that only add targets, no component or anything else since that will be the job of the parent app.

**Pull request closed without merging**